pkg/minikube/assets/addons.go

```diff
@@ -134,7 +134,7 @@ var Addons = map[string]*Addon{
 		MustBinAsset(addons.DashboardAssets, "dashboard/dashboard-secret.yaml", vmpath.GuestAddonsDir, "dashboard-secret.yaml", "0640"),
 		MustBinAsset(addons.DashboardAssets, "dashboard/dashboard-svc.yaml", vmpath.GuestAddonsDir, "dashboard-svc.yaml", "0640"),
 	}, false, "dashboard", "kubernetes", map[string]string{
-		"Dashboard":      "kubernetesui/dashboard:v2.3.1@sha256:ec27f462cf1946220f5a9ace416a84a57c18f98c777876a8054405d1428cc92e",
+		"Dashboard":      "kubernetesui/dashboard:v2.5.0@sha256:ec27f462cf1946220f5a9ace416a84a57c18f98c777876a8054405d1428cc92e",
```

Comment:
Hi! I think I have to change the hash here, but the hash for v2.3.1 is different from the one I checked on dockerhub. Could you tell me where can I find it?

Reply:
You can get the digest(kubernetesui/dashboard:v2.3.1@sha256:ec27f462cf1946220f5a9ace416a84a57c18f98c777876a8054405d1428cc92e) by `docker inspect kubernetesui/dashboard:v2.3.1 --format='{{index .RepoDigests}}'`.
Note that you need to pull the image(`kubernetesui/dashboard:v2.3.1`) to local, in advance.

Reply:
Thank you so much for your help!!